admin_manual/configuration_database/mysql_4byte_support.rst

```diff
@@ -52,7 +52,12 @@ The result should look like this::
     ALTER DATABASE nextcloud CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci;
 
 5. Set the ``mysql.utf8mb4`` config to true in your config.php::
-
+>>>>>>>>>>>>>>>>>>>>
```

Comment:
This does not work in rst.

Anyway, for me the command still works, even with maintenance mode enabled:
```
$ occ config:system:set 'mysql.utf8mb4' --type boolean --value="true"
Nextcloud is in maintenance mode, no apps are loaded.
Commands provided by apps are unavailable.
System config value mysql.utf8mb4 set to boolean true
```